sqlmesh/core/context.py

```diff
@@ -1846,15 +1848,77 @@ def create_external_models(self, strict: bool = False) -> None:
                 strict=strict,
             )
 
+    def get_sensitive_fields(self) -> Set[str]:
+        """Returns a set of common sensitive field names"""
+        return {
+            "password",
+            "secret",
+            "key",
+            "token",
+            "api_key",
+            "private_key",
+            "access_token",
+            "refresh_token",
+            "auth_token",
+            "credentials",
+            "client_secret",
+            "certificate",
+            "ssh_key",
+        }
+
+    def is_sensitive_field(self, field_name: str, sensitive_fields: Set[str]) -> bool:
```

Comment:
Shouldn't be a method. Let's move this to utils along with the list of sensitive fields.